imports/collections/topic.schema.js

```diff
@@ -32,6 +32,8 @@ export const TopicSchema = SchemaClass.create({
 
 if (Meteor.isServer) {
     Meteor.publish('topics', function (meetingSeriesId) {
```

Comment:
Please rename `meetingSeriesId` to reflect that it may also be an array. e.g., `meetingSeriesIdOrArray`